File: MoM.Module/Dtos/UserDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MoM.Module.Dtos
{
    public partial class UserDto
    {
        public string id { get; set; }
        public string userName { get; set; }
        public string email { get; set; }
        public bool emailConfirmed { get; set; }
        public int accessFailedCount { get; set; }
        public string concurrencyStamp { get; set; }
        public bool lockoutEnabled { get; set; }
        public DateTimeOffset? lockoutEnd { get; set; }
        public string phoneNumber { get; set; }
        public bool phoneNumberConfirmed { get; set; }
        public bool twoFactorEnabled { get; set; }
        public List<RoleDto> roles { get; set; }
    }
}

File: MoM.Web/app/dtos/UserDto.ts
import {RoleDto} from "./RoleDto";

export interface UserDto {
    id: string;
    userName: string;
    email: string;
    emailConfirmed: boolean;
    accessFailedCount: number;
    concurrencyStamp: string;
    lockoutEnabled: boolean;
    lockoutEnd: Date;
    phoneNumber: string;
    phoneNumberConfirmed: boolean;
    twoFactorEnabled: boolean;
    roles: RoleDto[];
}
